### ISSUE TITLE
Documentation Request - Add Search Editor to User Guide

### ISSUE THREAD

**Issue by KyleMit** [microsoft/vscode-docs#3947]
While the search editor has found its way into many of the release notes within 2020 - I couldn't find a one stop shop to tell me more about it within the evergreen documentation.

Here's some of the respective update posts

* [**January 2020** (v1.42)](https://code.visualstudio.com/updates/v1_42#_search-editor)
* [**February 2020** (v1.43)](https://code.visualstudio.com/updates/v1_43#_search-editors)
* [**May 2020** (v1.46)](https://code.visualstudio.com/updates/v1_46#_search-editor)
* [**June 2020** (v1.47)](https://code.visualstudio.com/updates/v1_47#_new-search-editor-command-arguments)
* [**July 2020** (v1.48)](https://code.visualstudio.com/updates/v1_48#_search-editor)

Maybe it could work it's way into the [User Guide under Search Across Files](https://code.visualstudio.com/docs/editor/codebasics?source=post_page#_search-across-files)?

**Comment by joey2031** [comment]
Hello I am new to the project and is trying to make my first contribution, after talking to some people it seems like this will be a good start for me. Can you please provide more information on what I need to do? Should I try to write something myself? Or copy paste elements from the update posts/summarize them? Thank you.

**Comment by gregvanl** [comment]
Hi @joey2031 This would be a new section in an existing topic. Probably several paragraphs, with some screenshots, describing the features, why it is valuable, and how to use it. 
To get started, I would first learn about the feature by trying it out and reading the release notes entries listed above.
Once you feel you understand the feature, a good practice is to try describing/demoing the feature to a few colleagues. Once you can talk about the feature comfortably, then you are ready to put together an outline.
Feel free to post your proposed outline in this issue for a quick review before you start an actual PR.

**Comment by joey2031** [comment]
> Hi @joey2031 This would be a new section in an existing topic. Probably several paragraphs, with some screenshots, describing the features, why it is valuable, and how to use it. To get started, I would first learn about the feature by trying it out and reading the release notes entries listed above. Once you feel you understand the feature, a good practice is to try describing/demoing the feature to a few colleagues. Once you can talk about the feature comfortably, then you are ready to put together an outline. Feel free to post your proposed outline in this issue for a quick review before you start an actual PR.

I'll start working on it, see what I can come out with and keep you updated. No matter what happens seems like a good opportunity to learn. Thank you! :)

**Comment by joey2031** [comment]
So I started working on this issue and this is what I have so far: https://github.com/joey2031/vscode-docs/blob/3947-searchEditorDoc/docs/editor/codebasics.md#search-editor 
There are some points I brought up (see my bolded comments). I still think it could be a bit more clear, and is also not sure if I listed all the search editor command arguments (I couldn't find anymore but you never know). Anyways before I continue I thought it would be a good idea to someone to look at this to receive some feedback and take it from there. Thank you.
